backend/core/src/migration/1654190736229-new-accessibility-features.ts

```diff
@@ -0,0 +1,28 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class newAccessibilityFeatures1654190736229 implements MigrationInterface {
+    name = 'newAccessibilityFeatures1654190736229'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "listing_features" ADD "barrier_free_unit_entrance" boolean`);
+        await queryRunner.query(`ALTER TABLE "listing_features" ADD "lowered_light_switch" boolean`);
+        await queryRunner.query(`ALTER TABLE "listing_features" ADD "barrier_free_bathroom" boolean`);
+        await queryRunner.query(`ALTER TABLE "listing_features" ADD "wide_doorways" boolean`);
+        await queryRunner.query(`ALTER TABLE "listing_features" ADD "lowered_cabinets" boolean`);
+        await queryRunner.query(`ALTER TABLE "listings" DROP COLUMN "marketing_season"`);
```

Comment:
Don't think we want to drop marketing season?

Reply:
good catch, not sure why the auto gen for migrations pulled that in, got it removed!